### PR TITLE
Add doc about RBAC

### LINKED ISSUE #2199: Cloud: Add doc about roles
### Related code PR

_No response_

### Which part(s) of the docs might be affected or should be updated? And how?

Add doc about roles

### Reference

_No response_

### PR DESCRIPTION
**Pull request by WanYixian**
## Info

- **Description**

    Add doc about RBAC

- **Related doc issue**
  
  Resolves https://github.com/risingwavelabs/risingwave-docs/issues/2199

## For reviewers

- **Preview**

  - [ Paste the preview link to the updated page(s) here. Edit this item after the preview site is ready. To find the updated pages, scroll down to locate and open the Amplify preview link and select the **dev** version of the documentation. ]

- **Key points**

  - [ Parts that may need revision or extra consideration. ]

## Before merging

- [ ] I have checked the doc site preview, and the updated parts look good.

- [ ] I have acquired the approval from the owner (and optionally the reviewers) of the code PR and at least one tech writer (`emile-00`, `hengm3467`, & `WanYixian`).

### CODE REVIEW

**Comment by WanYixian** [comment]
@xuhui-lu Hi Xuhui, please help add more input about rolebinding in the doc, thanks!

**Comment by xuhui-lu** [comment]
> @xuhui-lu Hi Xuhui, please help add more input about rolebinding in the doc, thanks!

I drafted a simple doc to describe it. Feel free to comment on the doc if necessary!
https://www.notion.so/risingwave-labs/Role-Bindings-Management-25bd2ead411f41a8b171d437d80b2127?showMoveTo=true&saveParent=true

**Comment by WanYixian** [comment]
Doc updated. See [preview](https://pr-2320.d2fbku9n2b6wde.amplifyapp.com/cloud/organization-rbac/). PTAL, thanks! @xuhui-lu @wyhyhyhyh

File: cloud/organization-rbac.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
+---
+id: organization-rbac
+title: Role-based access control (RBAC)
+description: Support role-based access control for cloud users.
+slug: /organization-rbac
+---
+
+In RisingWave Cloud, Role-based access control (RBAC) system empowers organizations to precisely manage access permissions based on their roles. Among RisingWave Cloud users in the organization, each user is associated with a pre-defined role. The organization administrator will have permission to edit the roles of all the members, whereas other members will have access to different subsystems depending on their roles.
```
Comment (wyhyhyhyh) [inline cloud/organization-rbac.md:8]: Is it "one pre-defined role"? cc @xuhui-lu

File: cloud/organization-rbac.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+
+Service accounts and users have distinct purposes:
+
+- Service accounts: Typically represents an application. It accesses RisingWave Cloud resources on the application's behalf, using API keys for authentication.
+
+- Users: Typically represents an individual user who interacts with resources via the user interface.
+
+## Role permissions and limitations
+
+Below are permissions and limitations for roles to ensure that each service account or user has appropriate access tailored to their responsibilities.
```
Comment (wyhyhyhyh) [inline cloud/organization-rbac.md:20]: Is a table view better than plain texts in this section?

**Review by wyhyhyhyh — APPROVED** [review]
LGTM in general

File: cloud/organization-rbac.md (hunk 1 above)
Comment (xuhui-lu) [reply cloud/organization-rbac.md:8]: I think it refers to the read-only role OrganizationMember, it is more like a default role associated with every org member.

**Comment by xuhui-lu** [comment]
The service accounts and users tab is more like describing the principal which rbac works for. How about naming it to be principal.

**Pull request merged**